square.py

```diff
@@ -0,0 +1,61 @@
+# We have a set of points described by (x, y) coordinates in the two dimensional space.
+# Create a function, that calculates coordinates of the smallest possible square (coord and size) that fits all of them.
+# Two sides of the square are expected to be parallel to the x-axis
+# Program prints results on console and draws points and square in a separate window
+
+import json
+import argparse
+import matplotlib.pyplot as plt
+
+# Uploading points from JSON file
+parser = argparse.ArgumentParser()
+parser.add_argument("-f", "--filename", required=True, type=str)
+args = parser.parse_args()
+with open(args.filename) as json_data:
```

Comment:
Try to execute that program with the file name that does not exist.
You need to start thinging not only the "goldem path", but also errors / exceptions.